### ISSUE TITLE
Disable certain blocks?

### ISSUE THREAD

**Issue by NadeSwiiv** [marcelo-mason/PreciousStones#972]
I would like to use only the city protection and the proximity mine. Is there a way to completely disable the rest of the special blocks, so I only have those on the server.

**Comment by marcelo-mason** [comment]
Yes, simply delete them from the force-field-blocks list in the config.

**Comment by NadeSwiiv** [comment]
Thank you! much appriciated!
